src/main/java/com/github/susom/dbgoodies/etl/Etl.java

```diff
@@ -308,9 +312,38 @@ public void start() throws Exception {
      */
     @CheckReturnValue
     SaveAsBigQuery fetchSize(int nbrRows) {
-      fetchSize = nbrRows;
+      this.fetchSize = nbrRows;
       return this;
     }
+
+    /**
+     * configure number of upload workers
+     */
+    @CheckReturnValue
+    SaveAsBigQuery workerNumber(int workerNum) {
+      this.workerNumber = workerNum;
+      return this;
+    }
+
+    /**
+     * configure batch size of each BigQuery insert
+     */
+    @CheckReturnValue
+    SaveAsBigQuery bigQueryBatchSize(int batchSize) {
+      this.batchSize = batchSize;
+      return this;
+    }
+
+    /**
+     * add labels to BigQuery table
+     */
+    @CheckReturnValue
+    SaveAsBigQuery bigQueryLabel(Map<String, String> labels) {
+      this.labels = labels;
+      return this;
+    }
```

Comment:
Might be more convenient to `withLabel(String)` and accumulate them rather than replacing the entire map (so stacked calls overwrite).

Reply:
Replacing entire map is by design. I can change the name to bigQueryLabels if it helps.

Reply:
What is the rationale for that design? As a builder, I can think of additive use cases, but when would I need/want to overwrite previous values? And the code will be cleaner without the collections.

Reply:
Ok, sounds good. If you make those `addLabel()` and `withLabels()` it might make the distinction a little clearer ("bigQuery" prefix doesn't add anything meaningful, IMO).

Reply:
good point. just made the change.